MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -923,7 +922,7 @@ private void processPrisonerCapture(List<TestUnit> unitsToProcess) {
                 }
                 if (en.hasWorkingMisc(MiscType.F_COMMAND_CONSOLE)) {
                     for (Person p : crew) {
-                        if (p != commander && p != driver) {
+                        if (!p.equals(commander) && !p.equals(driver)) {
```

Comment:
I think reference equality is okay here actually. We wouldn't want two instances with the same UUID. In any case this is safe for now.

Reply:
It is okay for now, but I'd rather future proof and implement it properly with equals comparisons.